modules/mono/editor/bindings_generator.cpp

```diff
@@ -309,7 +309,16 @@ String BindingsGenerator::bbcode_to_xml(const String &p_bbcode, const TypeInterf
 				// We do not declare theme_items in any way in C#, so there is nothing to reference
 				_append_xml_undeclared(xml_output, link_target);
 			} else if (link_tag == "param") {
-				_append_xml_undeclared(xml_output, snake_to_camel_case(link_target, false));
+				if (p_is_signal) {
+					// Documentation in C# is added to an event, not the delegate itself
+					// As such, we treat these parameters as codeblocks instead
+					// See: https://github.com/godotengine/godot/pull/65529
+					_append_xml_undeclared(xml_output, snake_to_camel_case(link_target, false));
+				} else {
+					xml_output.append("<paramref name=\"");
+					xml_output.append(snake_to_camel_case(link_target, false));
+					xml_output.append("\"/>");
+				}
```

Comment:
We could probably move this to an `_append_xml_param` method, since it seems we do that for all the others.